watchmal/dataset/h5_dataset.py

```diff
@@ -46,15 +46,15 @@ def __init__(self, h5_path, is_distributed):
     def initialize(self):
         self.h5_file = h5py.File(self.h5_path, "r")
 
-        self.event_ids  = np.array(self.h5_file["event_ids"])
-        self.root_files = np.array(self.h5_file["root_files"])
+#        self.event_ids  = np.array(self.h5_file["event_ids"])
+#        self.root_files = np.array(self.h5_file["root_files"])
         self.labels     = np.array(self.h5_file["labels"])
-        self.positions  = np.array(self.h5_file["positions"])
-        self.angles     = np.array(self.h5_file["angles"])
-        self.energies   = np.array(self.h5_file["energies"])
-        if "veto" in self.h5_file.keys():
-            self.veto  = np.array(self.h5_file["veto"])
-            self.veto2 = np.array(self.h5_file["veto2"])
+#        self.positions  = np.array(self.h5_file["positions"])
+#        self.angles     = np.array(self.h5_file["angles"])
+#        self.energies   = np.array(self.h5_file["energies"])
```

Comment:
are these not necessary passthrough anymore?

Reply:
They never were necessary for classification actually. Have left commented because other branches (regression and segmentation) do use them so when we merge and resolve the conflict we can make it load the things that are needed and not those that aren't.